tests/ecosystem/upgrade/test_upgrade_ocp.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from ocs_ci.ocs import ocp
+from ocs_ci.framework import config
+from ocs_ci.utility.utils import TimeoutSampler, get_latest_ocp_version
+from ocs_ci.framework.testlib import ManageTest
+from ocs_ci.ocs.cluster import CephCluster, CephHealthMonitor
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: add image type validation (ga to ga , nightly to nightly, newer than current etc.)
+
+
+class TestUpgradeOCP(ManageTest):
+    """
+    1. check cluster health
+    2. check OCP version
+    3. perform OCP upgrade
+    4. check all OCP ClusterOperators
+    5. check OCP version
+    5. monitor cluster health
+    """
+
+    def test_upgrade_ocp(self):
+        """
+        Tests OCS stability when upgrading OCP
+
+        """
+
+        ceph_cluster = CephCluster()
+        with CephHealthMonitor(ceph_cluster):
+
+            target_image = config.UPGRADE['ocp_upgrade_version']
+            if not target_image:
+                ocp_channel = config.UPGRADE['ocp_channel']
+                ocp_upgrade_version = get_latest_ocp_version(channel=ocp_channel)
+                ocp_arch = config.UPGRADE['ocp_arch']
+                target_image = f"{ocp_upgrade_version}-{ocp_arch}"
```

Comment:
Please go with approach from comment above so this line 38 and 39 should be one indent left.